src/branch.hpp

```diff
@@ -0,0 +1,12 @@
+#pragma once
+#include <string>
+class Branch {
```

Comment:
Entre la instrucción `#pragma once`, los _include_ y el inicio de la clase siempre es mejor poner una línea vacía. ¿Podrías hacer esto mismo en todos los archivos también?

```suggestion
#pragma once

#include <string>

class Branch {
```